### DISCUSSION TITLE
Step Forward goes 30s instead of 5s

### DISCUSSION THREAD

**Discussion by meckahi** [iina/iina#5036] [Q&A]
Under Playback menu, it says Right and Left arrow keys are intended to skip 5s.  It is skipping 30s instead.  I cannot find a setting to change this.

**Comment by low-batt** [comment]
[This comment](https://github.com/iina/iina/issues/4766#issuecomment-1883901903) in issue #4766 explains why this is the case and at the end of the comment shows how to change the behavior.

We have been discussing adding a setting to directly control this behavior.

**Comment by meckahi** [comment]
Thank you, I didn't find that thread before - adding hr-seek in the advanced menu as suggested here worked : [https://github.com/iina/iina/issues/4766#issuecomment-1883901903](url)
